packages/react-scripts/template/README.md

```diff
@@ -2432,7 +2432,8 @@ If, when deploying, you get `/dev/tty: No such a device or address` or a similar
 ### [Heroku](https://www.heroku.com/)
 
 Use the [Heroku Buildpack for Create React App](https://github.com/mars/create-react-app-buildpack).<br>
-You can find instructions in [Deploying React with Zero Configuration](https://blog.heroku.com/deploying-react-with-zero-configuration).
+You can find instructions in [Deploying React with Zero Configuration](https://blog.heroku.com/deploying-react-with-zero-configuration).<br>
+Getting started with [A Rock Solid, Modern Web Stack—Rails 5 API + ActiveAdmin + Create React App on Heroku](https://blog.heroku.com/a-rock-solid-modern-web-stack).
```

Comment:
Hmm, I'm not sure if this flows well by just adding it after these lines.

Can we make a nice transition for the reader? This is a separate guide.

Reply:
Hi @Timer, thanks for the feedback 💯.

What would make for a nicer transition for the reader? I've looked at all the mentions of "Heroku" in this user guide and this seemed the most fitting. Is there another place that's better that you can point me to?